### DISCUSSION TITLE
Error downloading from site binged.in

### DISCUSSION THREAD

**Discussion by Darken5** [aclap-dev/video-downloadhelper#469] [Help and Q&A]
### Your question

Trying to download from https://binged.in

specific movie https://binged.in/watch/movie/14522

### Addon & CoApp version and details

{
  "version": "9.0.1.9a1",
  "target": "mozilla",
  "channel": "beta",
  "page_url": "https://binged.in/watch/movie/14522",
  "error": {
    "error": "coapp_failure",
    "details": "SideDownload error: : [hls @ 0000020b6f0f8d00] Error when loading first segment 'https://up.vidstream.cloud/_v11/217efc76efa0ab30dead9f017580c6d011c412122b5ee83afbd26fa199b839883ec4dfb4a635cfe16f74c2c09dfcff6a6040b13d63e6971191037e6dbdbc3f1c9dcb2e12a6cc5808e2cd0330e36f0125405a8b502b8c0b9238818359892084a8d294dab2ead497c1e5708b13c92e9037253d10bab40344559bc103193e64b91b86414621f15b8b547fae49c67668dc20/720/seg-1-v1-a1.jpg'\r\n[in#0 @ 0000020b6f0f8b80] Error opening input: Invalid data found when processing input\r\nError opening input file https://vidsrc.pro/api/proxy/viper/_v11/217efc76efa0ab30dead9f017580c6d011c412122b5ee83afbd26fa199b839883ec4dfb4a635cfe16f74c2c09dfcff6a6040b13d63e6971191037e6dbdbc3f1c9dcb2e12a6cc5808e2cd0330e36f0125405a8b502b8c0b9238818359892084a8d294dab2ead497c1e5708b13c92e9037253d10bab40344559bc103193e64b91b86414621f15b8b547fae49c67668dc20/720/index.m3u8.png?base=r.pollllop.com.\r\nError opening input files: Invalid data found when processing input\r\n",
    "id": "coapp_failure_626926b3-3cb8-4b48-9a4c-db771418a514",
    "downloadable_id": "downloadable_2011749734293796"
  }
}

**Comment by GrampaWildWilly** [comment]
I do not work for VDH.  I'm just another user.

I am running Windows 11 Pro 64-bit version 23H2 build 22631.3447, Firefox 125.0.3 64-bit, licensed VDH 9.0.1.9a1 beta, CoApp 2.0.19.  You did not provide all of this information.  It turns out this isn't a roadblock here, as will become apparent in a moment.  But usually, you should supply all of it.

As you can see in this image, I got the same error you did:

![#01](https://github.com/aclap-dev/video-downloadhelper/assets/75588964/993c209f-2071-4fe6-9ac9-04bbdbc34421)

You can also see in that image that I opened the Network Monitor & found an HLS master manifest.  Rather ominously, the right half of the Network Monitor where a Response would normally be shown is empty.  Nevertheless, I ran ffprobe on it & got these results:

[ffprobe.txt](https://github.com/aclap-dev/video-downloadhelper/files/15269029/ffprobe.txt)

403 Forbidden means you must have authorization to access the file in question.  And what exactly is that authorization?  Nobody outside the web admins on that site could possibly know.  This is something I've encountered on occasion.  It's like a stealth DRM protection.  Instead of using actual DRM, the web admins have made their manifests inaccessible to the unwashed public like you & me.  And VDH.  The web page performs some sort of magic to send the necessary authorization back to the server when you click the Play button in the player on the web page.  I clicked play & it did start playing the movie.  But I stopped it after a few seconds because I wasn't interested in sitting & watching it.  But you're not going to download this.  Too bad.  We're out of luck here.  I haven't investigated the rest of the site.  It's quite possible that some content is protected like this & some is not.  It looks to me like VDH would have no problem downloading unprotected content.  But this one is protected so forget it.
